.github/workflows/zizmor.yml

```diff
@@ -29,10 +29,17 @@ jobs:
         with:
           python-version: 3.12
 
+      - name: Set Up UV
+        id: setup_uv
+        uses: astral-sh/setup-uv@22695119d769bdb6f7032ad67b9bca0ef8c4a174
+        with:
+          version: "latest"
+          python-version: "3.12"
```

Comment:
I just noticed that this is 3.12 and the others are 3.11

Reply:
only thing is that it matches the Airflow Python version, but unsure of the consequences of updating when it comes to the packages we build (compute profiles and standard operator)

I probably set it to 12 because its running zizmor, which could maybe even be split out into its own uv group